newspaper/cli.py

```diff
@@ -221,10 +221,12 @@ def csv_string(article_dict: dict) -> str:
 
         output = article.to_json(as_string=args.output_format == "json")
         if args.output_format == "json":
+            assert isinstance(output, str)
```

Comment:
Do you think that maybe a test case for article.to_json would be better than an assert here?
My thoughts: only if to_json is buggy, this would trigger. For the enduser it would not mean a lot to get an assertion error here.

Reply:
I've added some overload bits (see https://stackoverflow.com/a/76302414/320546 for explanation) to `to_json` to make it have a more explicit link between the params and output, and so have removed these.